src/js/components/Calendar/Calendar.js

```diff
@@ -692,9 +698,15 @@ const Calendar = forwardRef(
               }}
             />
             <Button
-              a11yTitle={nextMonth.toLocaleDateString(locale, {
-                month: 'long',
-                year: 'numeric',
+              a11yTitle={format({
+                id: 'calendar.next',
```

Comment:
this id is specific to the onClick button announcement to a user that the calendar UI has moved to the next month. with your current implementation, if a user does not provide a messages prop, the button label is "Moved to {date}" and the user is no longer accurately informed when the UI changes when they click the button. this does not align with the expected screen reader experience. i'm proposing the following: 

- change the `default.json` language file to include new id values for the calendar:
````
"calendar": {
    "previousMove": "Moved to {date}",
    "previous": "Go to {date}",
    "nextMove": "Moved to {date}",
    "next": "Go to {date}"
},
````
- update the onClick announce format id to use `'calendar.nextMove'` instead of `'calendar.next'`
- update the onClick announce format id to use `'calendar.previousMove'` instead of `'calendar.previous'`

with these changes, the user experience is this:
- when navigating to the previous arrow button, the screen reader will read "Go to [previous month]"
- when navigating to the next arrow button, the screen reader will read "Go to [next month]"
- when the user presses the previous arrow button, the screen reader will announce "Moved to [previous month]"
- when the user presses the next arrow button, the screen reader will announce "Moved to [next month]"

let me know if you have questions, thank you!